nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -64,6 +64,8 @@
 
 - [Typesense](https://github.com/typesense/typesense), a fast, typo-tolerant search engine for building delightful search experiences. Available as [services.typesense](#opt-services.typesense.enable).
 
+- [Guix](https://guix.gnu.org), a functional package manager inspired by Nix is available to run with [services.guix](#opt-services.guix.enable).
```

Comment:
This should now be moved to `nixos/doc/manual/release-notes/rl-2405.section.md`.